conda-envs/environment-dev-py37.yml

```diff
@@ -13,6 +13,7 @@ dependencies:
 - fastprogress>=0.2.0
 - h5py>=2.7
 - ipython>=7.16
+- jax
```

Comment:
I am not sure we should add jax as a dependency. By that logic we should also add numpyro and blackjax...

The jaxtests workflow installs it manually

Reply:
Oh this was for documentation... Yeah it seems like we need to have an environment for docs... feel free to ignore my comment then

Reply:
I don't think you need them in the `dev` environments, only for `test` (until we add a docs env).

Reply:
We need for the docs to render properly I think. Is that so @OriolAbril?

Reply:
Are the docs using the `dev` environments?

Reply:
Yes, the python 3.8 dev one

Reply:
I see, shouldn't we have it use the test env instead? I guess if we start a doc env it doesn't really matter, but maybe they would end up identical?

Reply:
I think docs requires quite more dependencies than tests